libs/sidebar/use-cases/src/lib/ports/ui/sidebar.adapter.ts

```diff
@@ -1,6 +1,6 @@
 import { Injectable } from '@angular/core';
 
 @Injectable()
-export abstract class SidebarUiPort {
+export abstract class SidebarAdapter {
```

Comment:
What about `SidebarUiAdapter` just to be more explicit? Just a suggestion. 🤔

Reply:
same comment as above. But `UiAdapter` is also possible. lets discuss in the next session